File: function-viewer-api/src/main/java/xyz/cxc6922/functionviewer2/entity/dto/RestApiResult.java
package xyz.cxc6922.functionviewer2.entity.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class RestApiResult {
    private String msg;
    private Object info;
    private int code;

    public RestApiResult(Object info) {
        this.info = info;
    }
}

File: function-viewer-api/src/main/scala/xyz/cxc6922/functionviewer2/model/dto/RestApiResult.scala
package xyz.cxc6922.functionviewer2.model.dto

import scala.beans.BeanProperty

class RestApiResult {
  @BeanProperty var code : Int = 0
  @BeanProperty var msg : String = ""
  @BeanProperty var data : Object = _

  def this(data: Object) {
    this()
    this.data = data
  }

  def this(code: Int, msg: String) {
    this()
    this.code = code
    this.msg = msg
  }
}
